examples/simple/igraph_assortativity_nominal.c

```diff
@@ -0,0 +1,44 @@
+#include <igraph.h>
+#include <stdio.h>
+
+int main(){
+    igraph_t g;
+    igraph_integer_t nodes = 1000, types = 50;
+    igraph_vector_int_t node_type_vec;
```

Comment:
Since `g` and `node_type_vec` are used only in the loop, can you move their declarations there?